lib/node_modules/@stdlib/math/base/special/besselj0/lib/besselj0.js

```diff
@@ -129,6 +129,12 @@ var sc = [];
 * v = besselj0( 1.0 );
 * // returns 0.76519768655796649
 *
+* v = besselj0( Infinity );
```

Comment:
Slightly different in this context. We have been more relaxed where we allow `Number.POSITIVE_INFINITY`, but not `Infinity`.